test/models/active_hash_test.rb

```diff
@@ -0,0 +1,43 @@
+# Copyright © Mapotempo, 2021
+#
+# This file is part of Mapotempo.
+#
+# Mapotempo is free software. You can redistribute it and/or
+# modify since you respect the terms of the GNU Affero General
+# Public License as published by the Free Software Foundation,
+# either version 3 of the License, or (at your option) any later version.
+#
+# Mapotempo is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+# or FITNESS FOR A PARTICULAR PURPOSE.  See the Licenses for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Mapotempo. If not, see:
+# <http://www.gnu.org/licenses/agpl.html>
+#
+require './test/test_helper'
+
+module Models
+  class ActiveHashTest < Minitest::Test
```

Comment:
I find ActiveHashTest name and filename a bit misleading, because we are not testing ActiveHash itself, plus we may change the gem in the future but the tests of the model will stay. So maybe ModelTest?

Reply:
After close inspection I realized that both of these tests can be moved to `test/models/vrp_test.rb` (not the api but the models) because the tests related to Models are already there.

Reply:
After continuing the PR, I realized that we are splitting the `models/vrp_test.rb` as well... I am not sure if this split is the split we want though because all other tests are still related to vrp.. 

Still these two tests can go into VrpRejectionTest because we are refusing the vrp here.

Reply:
@fab-girard should we split `models/vrp_test.rb` from `test/models` section ?

Reply:
Yes as I explained you, the best practice is to have a lot of very small files instead of one big file. Rubocop default class length is 100 for remember! We have set it to 1000 because of too many offenses! There are many advantages:  easier to navigate in text editor, avoid git conflicts, structure code by features, etc...